euth/communitydebate/forms.py

```diff
@@ -1,6 +1,7 @@
 from django import forms
 
 from adhocracy4.categories import forms as category_forms
+from euth.offlinephases import widgets as offlinephase_widgets
```

Comment:
This is just a small comment/suggestion: the widget should be moved to contrib then, as it is used in two apps.